lib/async_hooks.js

```diff
@@ -61,13 +61,7 @@ async_wrap.setupHooks({ init,
 
 // Used to fatally abort the process if a callback throws.
 function fatalError(e) {
-  if (typeof e.stack === 'string') {
-    process._rawDebug(e.stack);
-  } else {
-    const o = { message: e };
-    Error.captureStackTrace(o, fatalError);
-    process._rawDebug(o.stack);
-  }
+  process._rawDebug(e.stack);
```

Comment:
> IMHO restore the weird `captureStackTrace` case, and please add it to the `module.exports = {`

I completely disagree, there is no good reason for stack trace to start in `fatalError` it should start where the error occurred. And `async_hooks` should definitely not export `fatalError`, as `fatalError` has nothing to do with `async_hooks`.